client/agenda/store.js

```diff
@@ -230,6 +230,38 @@ export const useAgendaStore = defineStore('agenda', {
 
       return lastEvent.id || null
     },
+    findNowEventId () {
+      const currentEventId = this.findCurrentEventId()
+      
+      if (currentEventId) return currentEventId
+
+      // if there isn't a current event then instead find the next event
+
+      const current = (this.nowDebugDiff ? DateTime.local().minus(this.nowDebugDiff) : DateTime.local()).setZone(this.timezone)
+
+      // -> Find next event after current time
+      let nextEvent = {}
+      for(const sh of this.scheduleAdjusted) {
+        if (sh.adjustedStart > current) {
+          // -> Use the first event of multiple events having identical times
+          if (nextEvent.start === sh.adjustedStart.toMillis()) {
+            continue
```

Comment:
Use break instead? No need to keep iterating, as events are already chronological.